gordo_components/builder/local_build.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+import io
+from typing import Iterable, Tuple, Union
+
+from sklearn.base import BaseEstimator
+
+from gordo_components.workflow.config_elements.normalized_config import NormalizedConfig
+from gordo_components.workflow.workflow_generator.workflow_generator import (
+    get_dict_from_yaml,
+)
+from gordo_components.builder.build_model import build_model
+
+
+def local_build(config_str: str) -> Iterable[Tuple[Union[BaseEstimator, None], dict]]:
+    """
+    Build model(s) from a bare Gordo config file locally.
+
+    This uses the much same process the the normal workflow generation and subsequent
+    deployment the normal Gordo deployment process makes. Should help developing locally
+    as well as giving a good indication that your config is valid for deployment
+    with Gordo.
```

Comment:
I assume you mean something in the ballpark of: "This uses much of the same process as the nornaml.."?

Reply:
me english best. Talk everyone at great me english knowledge.